### PR TITLE
Issue 34 debug

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**
Addressed the bugs in issue #34 
also: improved icon colors in project title

### CODE REVIEW

**Comment by jmurugan-fzj** [comment]
@SteffenBrinckmann I do the review until the steps to reproduce the issue is clear to me, as I mentioned in the parent bug!

File: pasta_eln/GUI/form.py [hunk 1]
```diff
@@ -352,13 +352,18 @@ def closeDialog(self, btn:QPushButton) -> None:
...
-    Event if user clicks button or users context menu
+    Event if user clicks button
 
     Args:
       command (list): list of commands
     """
     if isinstance(command[0], CommandMenu):
-      executeContextMenu(self, command)
+      if executeContextMenu(self, command):
+        print('I should repaint the image', len(self.doc))
```
Comment (jmurugan-fzj) [inline pasta_eln/GUI/form.py:362]: Is this print message needed, can't we use logging instead of print message?

**Review by jmurugan-fzj — APPROVED** [review]

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj I have read a small howto in the corresponding issue. You can try, per your convenience, it.
I will merge it, sometime later as I have to teach phd students now.

**Comment by jmurugan-fzj** [comment]
> @jmurugan-fzj I have read a small howto in the corresponding issue. You can try, per your convenience, it. I will merge it, sometime later as I have to teach phd students now.

@SteffenBrinckmann I have approved the PR already, please go ahead and merge!

**Pull request merged**